website/docs/docs/partnerships.mdx

```diff
@@ -154,15 +146,17 @@ HashiCorp Terraform has an extremely wide community of users and contributors, a
 
 Vendors who choose not to support their provider and prefer to make it a community-supported provider will no longer be listed as Partner.
 
+-> Terraform modules published by Partner: We now have the ability to tag partner published modules in the Terraform Registry. If you have a modules that you’d like to receive the “Partner” tag, please reach out to us at technologypartners@hashicorp.com
+
 ## Terraform Cloud Integrations
 
-Run tasks allow Terraform Cloud to execute tasks in external systems at certain points in the Terraform Cloud run lifecycle. Specifically, Terraform Cloud users can add and execute these tasks during the pre-plan (public beta) and post-plan run stages. To execute a run task, Terraform Cloud sends an API payload to the external system. This payload contains a collection of run-related information and a callback URL that the external system can use to send updates back to Terraform Cloud.
+Run tasks allow Terraform Cloud to execute tasks in external systems at certain points in the Terraform Cloud run lifecycle. Specifically, Terraform Cloud users can add and execute these tasks during the pre-plan (public beta), pre-apply (public beta) and post-plan run stages. To execute a run task, Terraform Cloud sends an API payload to the external system. This payload contains a collection of run-related information and a callback URL that the external system can use to send updates back to Terraform Cloud.
 
 The external system can then use this run information and respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the task's enforcement settings within a workspace.
 
-Partners who successfully complete the Terraform Cloud integration checklist obtain a Terraform Cloud badge. This signifies HashiCorp has verified the integration and the partner is a member of the HashiCorp Technology Partner Program.
+Partners who successfully complete the Terraform Cloud integration checklist obtain a Terraform Cloud badge. This signifies HashiCorp has validated the integration under ideal conditions and the partner is a member of the HashiCorp Technology Partner Program.
```

Comment:
```suggestion
Partners who successfully complete the Terraform Cloud integration checklist obtain a Terraform Cloud badge. This badge signifies HashiCorp validated the integration under ideal conditions and the partner is a member of the HashiCorp Technology Partner Program.
```

Suggest adding "badge" to avoid a "This" with no subject. Suggest changing "has validated" to "validated" to avoid a passive voice style issue.